rosetta-source/src/main/resources/cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/expectations.json

```diff
@@ -7,15 +7,15 @@
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex25-option-strategyComponentIdentifier.xml","excludedPaths":0,"externalPaths":97,"outstandingMappings":64,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[],"qualifiableObjectCount":0}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex26-fxswap-multiple-USIs.xml","excludedPaths":0,"externalPaths":64,"outstandingMappings":15,"validationFailures":8,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Swap","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex26-fxswap-multiple-USIs.xml","excludedPaths":0,"externalPaths":64,"outstandingMappings":15,"validationFailures":12,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Swap","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex27-flexible-term-forward.xml","excludedPaths":0,"externalPaths":44,"outstandingMappings":33,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[],"qualifiableObjectCount":0}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex29-fx-swap-with-multiple-identifiers.xml","excludedPaths":0,"externalPaths":78,"outstandingMappings":17,"validationFailures":8,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Swap","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex30-variance-swap.xml","excludedPaths":0,"externalPaths":57,"outstandingMappings":11,"validationFailures":10,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_ParameterReturnVariance","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex31-volatility-swap.xml","excludedPaths":0,"externalPaths":55,"outstandingMappings":12,"validationFailures":11,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_ParameterReturnVolatility","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex29-fx-swap-with-multiple-identifiers.xml","excludedPaths":0,"externalPaths":78,"outstandingMappings":17,"validationFailures":12,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Swap","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
```

Comment:
FX sample does not specify rate source, so required field (x2) `primaryFxSpotRateSource` is not set